src/components/fields/Email/Settings.tsx

```diff
@@ -0,0 +1,21 @@
+import { ISettingsProps } from "@src/components/fields/types";
+import { TextField } from "@mui/material";
+
+export default function Settings({ onChange, config }: ISettingsProps) {
+    return (
+        <>
+            <TextField
+                type="text"
+                label="Validation regex"
+                id="validation-regex"
+                value={config.validationRegex}
+                defaultValue="^[a-zA-Z0-9+_.-]+@[a-zA-Z0-9.-]+.[a-zA-z]{2,3}$"
+                fullWidth
+                onChange={(e) => {
+                    if (e.target.value === "") onChange("validationRegex")(null);
+                    else onChange("validationRegex")(e.target.value);
+                }}
```

Comment:
Because it’s set as `defaultValue` here, it appears as if the regex is already set. But it’s only set when the text field has its value changed, causing confusion.

Reply:
Got it.
That means we need to have a button that fills the regex expression with the default value triggering the onChange method.

Reply:
Hi @notsidney 
Are you expecting something like this?
![emailRegex](https://user-images.githubusercontent.com/16620150/203041334-433fc444-6cb0-4433-b712-665790d1e9d9.png)

Reply:
Hi @notsidney 

In the latest push https://github.com/rowyio/rowy/compare/main...RajGM:rowy:emailValidation_799

The standard regex appears as expected after clicking on the button, but it's not validating the email with regex.